README.md

```diff
@@ -28,4 +28,6 @@ npm install -g truffle ganache-cli
 
 truffle compile
 truffle migrate
-```
\ No newline at end of file
+```
+
+MIT License
```

Comment:
Consider adding more detailed documentation or comments in the README file, explaining the project's purpose, architecture, and any other relevant information.